0003-I-dapr-compose.md

```diff
@@ -0,0 +1,294 @@
+<checkout> ==APP==pinging dapr API
+<checkout> INFO[0000] enabled gRPC tracing middleware               app_id=checkout instance=Mukundans-MacBook-Pro.local scope=dapr.runtime.grpc.api type=log ver=1.9.3
+<order-proc> INFO[0000] started daprd              app_id=order-proc instance=Mukundans-MacBook-Pro.local scope=dapr.runtime.grpc.api type=log ver=1.9.3
+<order-proc> ==APP== starting the application
+<order-proc> ==APP== processing request 1
+<order-proc> ==APP== processing request 2
+<checkout> INFO[0000] started daprd               app_id=checkout instance=Mukundans-MacBook-Pro.local scope=dapr.runtime.grpc.api type=log ver=1.9.3
+<checkout> ==APP== processing request 2
+<order-proc> ==APP== processing request 3. request 3 calls dapr API
+<order-proc> INFO[0000] requst 3 is calls dapr API             app_id=order-proc instance=Mukundans-MacBook-Pro.local scope=dapr.runtime.grpc.api type=log ver=1.9.3
+<checkout> ==APP== processing request 3
+
+```
+
+The logs will be as shown above, it will be chaotic to see what application is in which state and so on.
+
+Instead of writing to STDOUT, each application and associated `daprd` process will write the logs to `{application dir}/.dapr/logs/{app id}/app_{datetime}.log`, `{application dir}/.dapr/logs/{app id}/daprd_{datetime}.log`.
+
+### Subcommands and flags for `compose` command in CLI
+
+It is proposed to have a subcommand `up` in `compose` command to _bring up_ all the applications and daprd sidecars. Later on an extension can be added to _shutdown all_ application and sidecars with a `down` subcommand.
+
+A `-f, --file` flag will be defined for passing in the location of the `compose` file to be used. If that is not given it defaults to `dapr.yaml` in the `pwd` (present working directory) or it will error out if that file cannot be found.
+
+For using the default file, the user will run
+
+```bash
+$ dapr compose up
+```
+The execution of the above command checks for `dapr.yaml` in `pwd` and uses it as the `compose` configuration file or errors out if that file is not found.
+
+For specifying a particular `compose` configuration file the user will run
+
+```bash
+$ dapr compose up -f <path/to/file.yaml>
+```
+
+The execution of the above command uses the `<path/to/file.yaml>` as the `compose` configuration file or errors out if that file is not found.
+
+
+## Feature lifecycle outline
+
+`dapr compose` will be a separate command in `dapr` CLI and is expected to coexist with `dapr run`.
+
+Compatibility with `dapr run` is expected to be maintained. But in certain cases there might be introduction of new behavior which might be _opt-in_ for `dapr run` whereas it might be _on by default_ for `dapr compose`.
+
+The expectation is for the behavior of `dapr compose` to be refined and stabilized over a series of releases starting from initial release in **v1.10**.
```

Comment:
I would advise to remove the expectations about the milestone for this feature and the actual feature proposal.